lib/views/lead/lead_schedule_view.dart

```diff
@@ -295,24 +290,24 @@ class _LeadScheduleViewState extends State<LeadScheduleView> {
                         enabledBorder: theme.inputDecorationTheme.border,
                         focusedBorder: theme.inputDecorationTheme.focusedBorder,
                         prefixIcon:
-                        const Icon(MdiIcons.gamepadCircleOutline, size: 24),
+                            const Icon(MdiIcons.gamepadCircleOutline, size: 24),
                       ),
                     ),
                   ),
                   Container(
                       margin: const EdgeInsets.only(top: 8),
-                      child: FxDateText(label: 'Schedule Date',
+                      child: FxDateText(
+                          label: 'Schedule Date',
                           initValue: DateTime.now(),
                           onDateChanged: (newDate) => scheduleDate = newDate,
-                          validator: (value) => _validate(value))
-                  ),
+                          validator: (value) => _validate(value))),
                   Container(
                       margin: const EdgeInsets.only(top: 8),
-                      child: FxTimeText(label: 'Schedule Time',
+                      child: FxTimeText(
+                          label: 'Schedule Time',
                           initValue: TimeOfDay.now(),
                           onTimeChanged: (newTime) => scheduleTime = newTime,
-                          validator: (value) => _validate(value))
-                  ),
+                          validator: (value) => _validate(value))),
                 ],
               ),
             ),
```

Comment:
Would you please revert back changes in this file and the next one.

Reply:
It's fixed